webapp/channels/src/components/new_search/search_box.tsx

```diff
@@ -326,7 +309,7 @@ const SearchBox = forwardRef(
                     searchType={searchType}
                     providerResults={providerResults}
                     selectedOption={selectedOption}
-                    showFilterHaveBeenReset={showFilterHaveBeenReset}
+                    showFilterHaveBeenReset={false}
```

Comment:
We should remove `showFilterHaveBeenReset` from `SearchBoxHints`, it's the only case where we needed it